### PR TITLE
Add Artax cookie leakage

### PR DESCRIPTION
**Pull request by kelunik**
* https://twitter.com/asyncphp/status/862033196824842246
* https://github.com/amphp/artax/commit/accdadaf78f7a43305c3a97d6a964bbc550a555d

### CODE REVIEW

File: amphp/artax/2017-05-09.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title:     Cookie leakage to wrong origins and non-restricted cookie acceptance
+link:      https://twitter.com/asyncphp/status/862033196824842246
+cve:       ~
+branches:
+    master:
+        time:     2017-05-09 19:42:02
+        versions: ['>=2', '<2.0.6']
+    1.x:
+        time:     2017-05-09 19:42:02
+        versions: ['>=1', '<1.0.6']
```
Comment (Ocramius) [inline amphp/artax/2017-05-09.yaml]: I assume pre-1.0 versions are also affected?

File: amphp/artax/2017-05-09.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+title:     Cookie leakage to wrong origins and non-restricted cookie acceptance
+link:      https://twitter.com/asyncphp/status/862033196824842246
```
Comment (stof) [inline amphp/artax/2017-05-09.yaml]: I would rather link to https://github.com/amphp/artax/releases/tag/v2.0.6 which gives some details about the vulnerability

File: amphp/artax/2017-05-09.yaml (hunk 1 above)
Comment (kelunik) [inline amphp/artax/2017-05-09.yaml]: I assume, too. Should I just drop `>=1`?

How should I add the `amp_v2` branch? Fixed that just now, but it doesn't have any released version yet.

File: amphp/artax/2017-05-09.yaml (hunk 2 above)
Comment (kelunik) [inline amphp/artax/2017-05-09.yaml]: Fixed locally, will push once @Ocramius responds to the other comment.

File: amphp/artax/2017-05-09.yaml (hunk 1 above)
Comment (Ocramius) [inline amphp/artax/2017-05-09.yaml]: > I assume, too. Should I just drop `>=1`?

I would suggest dropping the lower boundary, yes

 > How should I add the amp_v2 branch? Fixed that just now, but it doesn't have any released version yet.

Not relevant then - doesn't need to be mentioned

File: amphp/artax/2017-05-09.yaml (hunk 2 above)
Comment (kelunik) [inline amphp/artax/2017-05-09.yaml]: Pushed.

File: amphp/artax/2017-05-09.yaml (hunk 1 above)
Comment (kelunik) [inline amphp/artax/2017-05-09.yaml]: Changed. How are branch names handled? Do they actually matter? Because `master` will change to `2.x` soon.

Comment (Ocramius) [inline amphp/artax/2017-05-09.yaml]: They don't really matter that much, yeah.

File: amphp/artax/2017-05-09.yaml (hunk 2 above)
Comment (stof) [inline amphp/artax/2017-05-09.yaml]: looks like you have not pushed the updated code

Comment (kelunik) [inline amphp/artax/2017-05-09.yaml]: Indeed, forgot to `git add`.

File: amphp/artax/2017-05-09.yaml (hunk 1 above)
Comment (fabpot) [inline amphp/artax/2017-05-09.yaml]: Changing `master` to `2.x` is actually a good idea.

**Comment by kelunik** [comment]
@fabpot @Ocramius Could you merge?

**Comment by fabpot** [comment]
Can you change master to 2.x?

**Comment by kelunik** [comment]
Done.

**Pull request merged**